packages/schema-blocks/src/functions/presenters/SidebarWarningPresenter.ts

```diff
@@ -63,20 +64,26 @@ export function replaceVariables( issue: analysisIssue ): string {
  * @returns {sidebarWarning} Any analysis conclusions that should be in the footer.
  */
 function getAnalysisConclusion( validation: BlockValidation, issues: analysisIssue[] ): sidebarWarning {
-	if ( issues.some( issue => issue.result === BlockValidation.MissingBlock ||
+	// Show a red bullet when not all required blocks have been completed.
+	if ( issues.some( issue => issue.result === BlockValidation.MissingBlock && issue.status === BlockType.Required ||
 		issue.result === BlockValidation.MissingAttribute ) ) {
 		return {
-			text: __( "Not all required blocks are completed! No " + issues[ 0 ].parent +
+			text: __( "Not all required blocks have been completed! No " + issues[ 0 ].parent +
 				" schema will be generated for your page.", "wpseo-schema-blocks" ),
 			color: "red",
 		} as sidebarWarning;
 	}
 
+	// Show a green bullet when all required blocks have been completed.
+	const requiredIssues = issues.filter( issue => {
```

Comment:
issues zijn altijd ongewenst toch? nooit required.
```suggestion
	const requiredBlockValidations = issues.filter( issue => {
```

Reply:
I renamed it to `requiredBlockIssues`. Since it's not the issues that are required, but the blocks.